tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+        False -> if the expected MTBF is not same as the actual MTBF
+
+    """
+    track_available_data = get_pool_availability_status(client)
+    pool_data = track_available_data[pool_name]
+    log.info(
+        " Formula - Mean Time Between Failures (MTBF) = Total uptime of pool/No of Failures"
+    )
+    try:
+        total_up_time_pool = time_to_seconds(pool_data[0])
+        no_of_failures = int(pool_data[2])
+        expected_MTBF = time_to_seconds(pool_data[3])
+    except ValueError as e:
+        print(f"Error: {e}")
+        return 1
+    if no_of_failures == 0:
+        actual_MTBF_output = 0
+    else:
+        actual_MTBF_output = math.floor(total_up_time_pool / no_of_failures)
+    msg_output = f"Case 1: The MTBF actual output is {actual_MTBF_output}"
+    log.info(msg_output)
+
+    if actual_MTBF_output != expected_MTBF:
+        msg_err = (
+            f"Case 1: The MTBF actual output is{actual_MTBF_output} and expected is {pool_data[3]}, "
+            f"which are not same "
+        )
+        log.error(msg_err)
+        return False
+    return True
+
+
+def test_MTTR_value(client, pool_name):
+    """
+    Method is used to test the Mean Time To Recover(MTTR) value.
+    Formula - Mean Time To Recover(MTTR) = Total downtime of pool/No of Failures
+    Args:
+        client: Client object
+        pool_name : pool name
+    Returns:
+        True -> if the expected MTTR is same as the actual MTTR
+        False -> if the expected MTTR is not same as the actual MTTR
+
+    """
+    track_available_data = get_pool_availability_status(client)
+    pool_data = track_available_data[pool_name]
+    log.info(
+        " Formula - Mean Time To Recover(MTTR) = Total downtime of pool/No of Failures"
+    )
+    try:
+        total_down_time = time_to_seconds(pool_data[1])
+        no_of_failures = int(pool_data[2])
+        expected_MTTR = time_to_seconds(pool_data[4])
+    except ValueError as e:
+        print(f"Error: {e}")
+        return 1
+    if no_of_failures == 0:
+        actual_MTTR_output = 0
+    else:
+        actual_MTTR_output = math.floor(total_down_time / no_of_failures)
+    msg_output = f"Case 2: The MTTR actual output is {actual_MTTR_output}"
+    log.info(msg_output)
+
+    if actual_MTTR_output != expected_MTTR:
+        msg_err = (
+            f"Case 2: The MTTR actual output is{actual_MTTR_output} and expected is {pool_data[3]}, "
+            f"which are not same "
+        )
+        log.error(msg_err)
+        return False
+    return True
+
+
+def test_score_value(client, pool_name):
+    """
+    Method is used to test the score value
+    Formula - Score = MTBF/(MTBF+MTTR)
+        Args:
+            client: Client object
+            pool_name : pool name
+        Returns:
+            True -> if the expected score is same as the actual score
+            False -> if the expected score is not same as the actual score
+
+    """
+    track_available_data = get_pool_availability_status(client)
```

Comment:
same comment as line no 444. handle get_pool_availability_status o/p for all cases.

Reply:
Modified the logic.